libcst/metadata/tests/test_expression_context_provider.py

```diff
@@ -376,3 +376,26 @@ def test_for(self) -> None:
                 },
             )
         )
+
+    def test_class(self) -> None:
+        wrapper = MetadataWrapper(parse_module("class Foo: pass"))
```

Comment:
To make the test comprehensive, let's also test the name of base classes and some names in the class body.